gorapass/src/filter_models.py

```diff
@@ -42,11 +44,34 @@ def filter_hikes(cls, hike_models, filters):
         return hike_models
 
     @classmethod
-    def valid_hike_filters(cls, filters):
-        for model_filter in filters:
-            if set(model_filter.keys()) != ModelFilter.FILTER_KEYS:
-                return False
-            if not hasattr(Hikes, model_filter[ModelFilter.ATTRIBUTE_NAME]):
-                return False
-
-        return True
+    def validate_hike_selectors(cls, selectors):
+        for selector in selectors:
+
+            # Check that the keys for the selector object are correct
+            if set(selector.keys()) != ModelFilter.SELECTOR_KEYS:
+                expected = ModelFilter.SELECTOR_KEYS
+                actual = set(selector.keys())
+                message = f'Selector objects must include attributes {expected}, not {actual}'
```

Comment:
I agree. I'll make it a task for later, but right now I don't want to go too deep into this because things are still likely to change. I want to make sure we've settled on the format before we polish things.